src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/services/RadicleProjectService.java

```diff
@@ -86,16 +87,26 @@ public ProcessOutput getVersion(String path) {
         }
     }
 
-    public ProcessOutput track(GitRepository root, String reference, String nodeUrl) {
+    public ProcessOutput trackRepo(RadTrack.Repo repo) {
         List<String> args = new ArrayList<>();
         args.add("track");
-        if (!Strings.isNullOrEmpty(reference)) {
-            args.add(reference);
+        args.add(repo.rid());
+        if (!repo.scope().equals(RadTrack.Scope.NONE)) {
+            args.add("--scope");
+            args.add(repo.scope().name.toLowerCase());
         }
-        if (!Strings.isNullOrEmpty(nodeUrl)) {
-            args.addAll(List.of("--seed", nodeUrl, "--remote"));
+        return executeCommand(".", args, null);
+    }
+
+    public ProcessOutput trackPeer(RadTrack.Peer peer) {
+        List<String> args = new ArrayList<>();
+        args.add("track");
+        args.add(peer.nid());
+        if (!Strings.isNullOrEmpty(peer.alias())) {
+            args.add("--alias");
+            args.add(peer.alias());
         }
-        return executeCommand(root.getRoot().getPath(), args, root);
+        return executeCommand(".", args, null);
```

Comment:
I don't understand this change, why is the working directory set as `.`? We can't even be sure what `.` means across OSes.
Is tracking a peer an action _outside_ of a repo?

Reply:
yy you can track outside the repo thats why I set as directory `.`